src/view/screens/SavedFeeds.tsx

```diff
@@ -298,9 +299,10 @@ function ListItem({
         <FeedSourceCard
           key={feedUri}
           feedUri={feedUri}
-          style={[styles.noTopBorder, isPinned && {paddingRight: 8}]}
+          style={[isPinned && {paddingRight: 8}]}
           showMinimalPlaceholder
           showSaveBtn={!isPinned}
+          hideTopBorder={true}
```

Comment:
other styles here add a border, so we'll just turn it off for all of the `FeedSourceCard`s in this list.